orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTask.java

```diff
@@ -267,28 +264,24 @@ private MonitoredDeployStageData getStageContext(StageExecution stage) {
     return stage.mapTo(MonitoredDeployStageData.class);
   }
 
-  private String getRetrofitLogMessage(Response response) {
-    if (response == null) {
+  private String getErrorMessage(SpinnakerServerException spinnakerException) {
+    if (!(spinnakerException instanceof SpinnakerHttpException)) {
       return "<NO RESPONSE>";
     }
 
-    String body = "";
-    String status = "";
-    String headers = "";
+    SpinnakerHttpException httpException = (SpinnakerHttpException) spinnakerException;
 
     try {
-      status = String.format("%d (%s)", response.getStatus(), response.getReason());
-      body =
-          CharStreams.toString(
-              new InputStreamReader(response.getBody().in(), StandardCharsets.UTF_8));
-      headers =
-          response.getHeaders().stream().map(Header::toString).collect(Collectors.joining("\n"));
+      String body =
+          Optional.ofNullable(objectMapper.writeValueAsString(httpException.getResponseBody()))
+              .orElse("");
+      return String.format("headers: %s\nresponse body: %s", httpException.getHeaders(), body);
     } catch (Exception e) {
       log.error(
-          "Failed to fully parse retrofit error while reading response from deployment monitor", e);
+          "Failed to fully parse http response while reading response from deployment monitor", e);
```

Comment:
```suggestion
          "Exception occurred while parsing the http error response details", e);
```

Reply:
I have modified the log message to : https://github.com/spinnaker/orca/blob/e21a5cc06971cddde34d9b3ac59264b7aa31ea86/orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTask.java#L280C6-L280C91 . 
Replaced parse with read